Objektet Obstacle för maps nu tillagt

jkim931 opened the pull request:
Obstacle innehåller data för o kunna visa saker på mapfunktionen,
DatabaseHelper uppdaterad för att hantera det nya objektet, objektet
LocalObstacleID är en klass som används som fullösning till ORMLites
tillkortakommanden. Glöm inte att köra DatabaseConfigUtil när ni tagit ner
denna för att uppdatera databasen (och ominstallera)!

jkim931 committed:
Objektet Obstacle för maps nu tillagt

Obstacle innehåller data för o kunna visa saker på mapfunktionen,
DatabaseHelper uppdaterad för att hantera det nya objektet, objektet
LocalObstacleID är en klass som används som fullösning till ORMLites
tillkortakommanden. Glöm inte att köra DatabaseConfigUtil när ni tagit ner
denna för att uppdatera databasen (och ominstallera)!

JakobBacklund commented:
Appen kraschar inte och jocke verkar ha gjort det han påstår.

willedanielsson commented:
DEN E FIINN!

Rovanion commented on src/gov/polisen/ainappen/Obstacle.java:
```diff
@@ -0,0 +1,109 @@
+	@DatabaseField
+	int deviceID;
+	@DatabaseField(useGetSet = true, id = true)
+	String obstacleID;
```
Kan vi nämna strängen localObstacleID så att obstacleID är en int både på servern och appen? Henning skrev just om det så för case.

jkim931 committed:
Ändrade namn på Obstacle till MapPoint

Bättre klassnamn och variabelnamn, även databasetables!

jkim931 committed:
Minor bugfixes

Fixade bättre variabelnamn, metodnamn, + fixade en databasbugg som gjorde
att det kunde krascha ibland (automagiskt, vet inte hur tbh).
